src/Apps/Artist/routes.tsx

```diff
@@ -1,20 +1,63 @@
+import { hasSections as showMarketInsights } from "Apps/Artist/Components/MarketInsights/MarketInsights"
 import {
   ArtworkFilters,
   initialArtworkFilterState,
 } from "Components/v2/ArtworkFilter/ArtworkFilterContext"
 import { isDefaultFilter } from "Components/v2/ArtworkFilter/Utils/isDefaultFilter"
 import { paramsToCamelCase } from "Components/v2/ArtworkFilter/Utils/urlBuilder"
-import { Redirect, RouteConfig } from "found"
+import { Redirect, RedirectException, RouteConfig } from "found"
+import * as React from "react"
 import { graphql } from "react-relay"
 import { ArtistAppFragmentContainer as ArtistApp } from "./ArtistApp"
-import { ArticlesRouteFragmentContainer as ArticlesRoute } from "./Routes/Articles"
+import { hasOverviewContent } from "./Components/NavigationTabs"
 import { AuctionResultsRouteFragmentContainer as AuctionResultsRoute } from "./Routes/AuctionResults"
-import { CVRouteFragmentContainer as CVRoute } from "./Routes/CV"
-import {
-  ArtistOverviewQuery,
-  OverviewRouteFragmentContainer as OverviewRoute,
-} from "./Routes/Overview"
-import { ShowsRouteFragmentContainer as ShowsRoute } from "./Routes/Shows"
+import { OverviewRouteFragmentContainer as OverviewRoute } from "./Routes/Overview"
+import { WorksRouteFragmentContainer as WorksRoute } from "./Routes/Works"
+
+graphql`
```

Comment:
?

Reply:
This is needed in the `render()` function below in order to determine whether we need to redirect or not.